apps/style/lessons.scss

```diff
@@ -1 +1,22 @@
-@import "aniGif";
+@import 'color';
+@import 'aniGif';
+
+table {
+  tbody {
+    /**
+     * A pattern our curriculum authors often use is to include tables of
+     * slides, where in each row there's a cell with a screenshot of the slide
+     * and a cell with the description.
+     *
+     * Previously, they had been manually styling each screenshot with a
+     * border. To replicate this functionality, we say that every image which
+     * is an "only child" (ie, every image that's in a cell all by itself)
+     * should have a border. If at some point in the future this ends up
+     * styling images we _don't_ want to be styled, we could find a more
+     * precise way to implement this.
+     */
+    img:only-child {
+      border: 1px solid $black;
```

Comment:
this selector looks like it will affect the right things for now, but makes me a little nervous that it might start adding styles unexpectedly if something changed in the future (e.g. we add an image somewhere in our resources editor table on the lesson edit page). one idea for the future would be to add a css classname the outer divs of our react components which render markdown, and then scope any CSS styles specific to markdown to only take effect in divs with this classname.

Reply:
yeah, that sounds great! I could probably do that as a quick follow-up work item tomorrow